app/controllers/invoices_controller.rb

```diff
@@ -1,5 +1,5 @@
 class InvoicesController < ApplicationController
   def index
-
+    @invoices = InvoiceItem.where(item_id: Item.where(merchant_id: 1).pluck(:id))
```

Comment:
Either push this down into the InvoiceItem model as a class method and call it something like `invoices_for_merchant(merchant)` or create a refactor Issue for `app/controllers/invoices_controller.rb` for the `index` action.